Update Intro to 3D page for Godot 4.7 (#12257)
godotengine/godot-docs: tutorials/3d/introduction_to_3d.rst

Using Blender-style transform shortcuts
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> You can enable Blender-style shortcuts for translating, rotating and scaling nodes.

Reviewer: ```suggestion
You can enable Blender-style shortcuts for translating, rotating, and scaling nodes.
```

Preview environment and light
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

By default, any 3D scene that doesn't have a :ref:`class_WorldEnvironment`
node, or a :ref:`class_DirectionalLight3D`, will have

> a preview turned on for what it's missing to light the scene.

Reviewer: I believe "what" is the subject here. Also maybe this sounds better:
```suggestion
a preview of what is missing to light the scene.
```

Author: There's something about that phrasing that sounds awkward, I'll try and think of a better way to write this.

Reviewer 2: Maybe this:

> By default, any scene that doesn't have a WorldEnvironment or DirectionaLLight3D node will use an editor-only preview node to act in its place.
